pytrickle/frames.py

```diff
@@ -179,4 +244,235 @@ def with_monotonic_timestamps(cls, frames: List[AudioFrame], request_id: str, st
             corrected_frame = AudioFrame._from_existing_with_timestamp(frame, corrected_timestamp)
             corrected_frames.append(corrected_frame)
         
-        return cls(corrected_frames, request_id) 
\ No newline at end of file
+        return cls(corrected_frames, request_id)
+
+
+# Frame Processing Utilities
+# ===========================
+
+class FrameProcessor:
+    """Reusable frame conversion and processing utilities."""
+    
+    @staticmethod
+    def convert_trickle_to_av(frame: VideoFrame) -> av.VideoFrame:
```

Comment:
```suggestion
    def convert_trickle_video_to_av(frame: VideoFrame) -> av.VideoFrame:
```

Reply:
I removed FrameProcessor and moved these to their appropriate VideoFrame and AudioFrame classes as helper object methods https://github.com/livepeer/pytrickle/pull/8/commits/70a0ede3859a13661aa4f86560500e13dc8a4f89

This should eliminate the ambiguity over naming and reduce complexity